taker-frontend/src/components/Trade.tsx

```diff
@@ -338,7 +338,7 @@ function Quantity({ min, max, onChange, quantity, lotSize, isLong }: QuantityPro
                     </NumberInputStepper>
                 </NumberInput>
             </InputGroup>
-            <FormHelperText>How much do you want to buy or sell?</FormHelperText>
+            <FormHelperText>How many contract should your position contain?</FormHelperText>
```

Comment:
As usual, this was inspired by our giant friend BitMEX 😀.
Can you elaborate what was confusing about this?

<img width="336" alt="image" src="https://user-images.githubusercontent.com/224613/182258091-8655db35-e488-466e-8ea2-d9daa5f01bee.png">

Reply:
I'm not quite sure "how many contract should your position contain" means exactly. Firstly, should it be "contracts" (plural)? Secondly, I'm just not sure what "one contract" would mean.